classes/class-p4-campaign-exporter.php

```diff
@@ -41,12 +41,11 @@ public function single_post_export_data() {
 		 * Export multiple data
 		 */
 		public function single_post_export_bulk() {
-			if ( ! empty( $_GET['post_type'] ) &&
-			'campaign' == $_GET['post_type'] &&
+			if ( 'campaign' === filter_input( INPUT_GET, 'post_type', FILTER_SANITIZE_STRING ) &&
```

Comment:
You can use here the const that I added [here](https://github.com/greenpeace/planet4-master-theme/blob/planet-3517/classes/class-p4-post-campaign.php#L17)
`P4_Post_Campaign::POST_TYPE`